spec/models/user_spec.rb

```diff
@@ -55,4 +55,19 @@
       expect(user1).to be false
     end
   end
+
+  describe 'review notification' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.